src/routes/backoffice.ts

```diff
@@ -282,10 +282,11 @@ export const updatePersonHandler = async (req, res) => {
   }
 
   let questionSet = null;
-  const shouldGenerateQuestionSet = [
-    req.body.customer_vetting_status,
-    req.body.risk_classification_status,
-  ].includes(CustomerVettingStatus.INFORMATION_REQUESTED);
+  const shouldGenerateQuestionSet =
+    req.body.customerVettingStatus ===
+      CustomerVettingStatus.INFORMATION_REQUESTED ||
+    req.body.riskClassificationStatus ===
+      RiskClarificationStatus.INFORMATION_REQUESTED;
```

Comment:
why was it changed?

Reply:
request body has props in camel case, at least when set from panel. I'll check again

Reply:
Reference: 
https://github.com/kontist/mock-solaris/blob/f41cf09b50ea36a3a7abec510473712f236c864c/src/routes/backoffice.ts#L268